### PR TITLE
Enhances the package metadata (should help with #23)

### PR DESCRIPTION
**Pull request by outscale-age**
The goal of this PR is to help with resolution of #23, by adding more metadata information inside PKG-INFO. To this end, we add:
* Classifiers
* Long Description via README.md
* License short name
* More to-the-point package Homepage.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
-    url="http://www.outscale.com/",
+    description="Outscale API SDK and CLI",
+    long_description=read("README.md"),
+    long_description_content_type="text/markdown",
+    license='BSD',
+    # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
```
Comment (aubustou) [inline setup.py:31]: il faut aussi 3.6, 3.7, 3.8 et 3.9

File: setup.py [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+    """Reads a file from a sub directory.
+
+    :param parts: components of the path to file
+    :return: the file contents
+    """
+
+    # intentionally *not* adding an encoding option to open, See:
+    #   https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
+    here = os.path.abspath(os.path.dirname(__file__))
+    return codecs.open(os.path.join(here, *parts), 'r').read()
```
Comment (aubustou) [inline setup.py:15]: on a vraiment besoin de ça?
open.read() pose un problème avec notre README.rst?

File: setup.py (hunk 1 above)
Comment (outscale-age) [reply setup.py:17]: Fixed.

File: setup.py (hunk 2 above)
Comment (outscale-age) [reply setup.py]: You're perfectly right, today readme.md is an ASCII file, so we don't need to complicate things.

**Pull request merged**